packages/frontend-api/src/Model/Resolver/Customer/User/CurrentCustomerUserQuery.php

```diff
@@ -32,4 +35,18 @@ public function currentCustomerUserQuery(): CustomerUser
 
         return $currentCustomerUser;
     }
+
+    /**
+     * @return \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUser[]
+     */
+    public function customerUsersQuery(): array
+    {
+        $currentCustomerUser = $this->currentCustomerUser->findCurrentCustomerUser();
+
+        if ($currentCustomerUser === null) {
+            throw new UserWarning('No customer user is currently logged in.');
```

Comment:
UserWarning and userError should not be use due to lack of user error code. 
In ordersquery we use throw `new InvalidTokenUserMessageException();` I think this is the way

Reply:
![thisistheway](https://github.com/user-attachments/assets/7cdbfe97-275f-49ec-bb78-fabb51ed55ba)

That makes sense. Changed it. Also changed in `CurrentCustomerUserQuery::currentCustomerUserQuery`

However I would like to discuss it with the team further. I see different approaches in FE API authorization.

Reply:
* https://github.com/shopsys/shopsys/pull/3286/commits/de41ae653dfd5cadf03e0b117c0588b8e80aab2b
* https://github.com/shopsys/shopsys/pull/3286/commits/52382e94af92452d2c4bcdcf9d6ed4234185f711